data/json/recipes/recipe_deconstruction.json

```diff
@@ -284,7 +284,7 @@
     "difficulty": 0,
     "time": 500,
     "reversible": true,
-    "qualities": [ {"id":"HAMMER","level":1} ], 
+    "qualities": [ {"id": "HAMMER","level": 1} ], 
```

Comment:
I suppose the canonical form would be:
`"qualities": [ { "id": "HAMMER","level": 1 } ]`